src/table/utils/__tests__/copy-utils.spec.ts

```diff
@@ -6,30 +6,27 @@ describe('copyCellValue: ', () => {
   };
   let writeMock: (arg: string) => void;
   let elementClass: string;
-  let closestElementClass: string;
 
   const classList = {
     contains: (className: string) => className === elementClass,
   };
 
-  const closest = (className: string) => {
-    className === closestElementClass;
+  const closest = () => {
     return {
-      querySelector: jest.fn(() => ({
-        textContent: 'svg text',
-      })),
+      querySelector: () => ({
+        textContent: 'text',
+      }),
     };
   };
 
   beforeEach(() => {
     elementClass = '';
-    closestElementClass = '';
     evt = {
       target: {
         closest,
-        querySelector: jest.fn(() => ({
-          textContent: 'head text',
-        })),
+        querySelector: () => ({
+          textContent: 'text',
```

Comment:
very minor thing, but now you return the same value from `querySelector` and `closest().querySelector`, so you can't really tell what happened in the function. could change the other one to `'textFromClosest'` or whatever

Reply:
Ahh, true! I removed them blindly. 
It is fixed now.